src/py42/clients/securitydata.py

```diff
@@ -468,3 +469,18 @@ def destination_guid(self):
     def node_guid(self):
         """The GUID of the storage node containing the archive."""
         return self._node_guid
+
+
+def _escape_quote_chars_in_token(string):
+    """
+    The `nextPgToken` returned in Forensic Search requests with > 10k results is the eventId
+    of the last event returned in the response. Some eventIds have double-quote chars in
+    them, which need to be escaped when passing the token in the next search request.
+    """
+    unescaped_quote_pattern = r'[^\\]"'
+
+    return re.sub(
```

Comment:
Why do we need the `match.group()`, why can't just use the replace call on its own?

Reply:
Like, 

```python
x = "asdfsdf\"asdf\"asdf335425'"
x.replace('"', r'\"')
```

outputs `'asdfsdf\\"asdf\\"asdf335425\''`, wouldn't that suffice?

Reply:
Oh I see it is weird, it like adds an extra slash... ok.